docs/manual/developer_guide.adoc

```diff
@@ -1732,7 +1732,7 @@ yamlfile_value::
 ** *value* - the value to check.
 ** *type* (SimpleDatatypeEnumeration) - datatype for state's value_of, optional.
 ** *entity_check* (CheckEnumeration) - entity_check value for state's value_of, optional.
-** *pattern_check* - if set to `"true"`, the value is handled as a pattern match for state's value_of, optional.
+** *operation* (OperationEnumeration) - operation value for state's value_of, optional.
```

Comment:
I suggest mentioning that the default operation is "equals".

Reply:
Technically it would be empty (not set). As far as I can see, the document's default values are explained from the point of view of the template, i.e. what would be written into the DS by the template if parameter is omitted, not what OVAL would imply if nothing would be added there by the template.

Should we then also mention that `datatype` would be *string* by default, `entity_check` would be *all*, etc?

Reply:
Thanks for clarification that it's implicit OVAL value. Based on that, I think it isn't necessary to mention it.

It might improve the understanding for the template user, though.

Reply:
Added an explanation